code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -1,13 +1,14 @@
 /turf
 	//used for temperature calculations
-	var/thermal_conductivity = 0.05
+	var/thermal_conductivity = 0.005
 	var/heat_capacity = 1
 	var/temperature_archived
 
 	///list of open turfs adjacent to us
 	var/list/atmos_adjacent_turfs
 	///bitfield of dirs in which we are superconducitng
 	var/atmos_supeconductivity = NONE
+	var/is_openturf = FALSE
```

Comment:
What is this var for?

Reply:
I believe it's read by extools
One of the downsides of said injection is some shit is much less transparent.